great_expectations/validator/validator.py

```diff
@@ -703,14 +701,6 @@ def compute_metrics(
             )
 
         resolved_metrics: Dict[Tuple[str, str, str], Any] = {}
-        # noinspection PyUnusedLocal
-        aborted_metrics_info: Dict[
-            Tuple[str, str, str],
-            Dict[str, Union[MetricConfiguration, List[Exception], int]],
-        ] = self.resolve_validation_graph(
-            graph=graph,
-            metrics=resolved_metrics,
-        )
```

Comment:
I think we still need to run `self.resolve_validation_graph()` here even if not using the `aborted_metrics_info`. Instead you can replace `aborted_metrics_info` with `_` to throw away the value.

Reply:
return value not required, but the call still is! good catch